### PR TITLE
[Keyboard] Update Iris readme

### PR DESCRIPTION
**Pull request by dlrobertson**
## Description

Update the Iris keyboard documentation to include instructions for flashing DFU bootloaders.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage). (*NB:* Tested rev3... I don't have an Elite-C, but the docs for it says it has a DFU bootloader)

CC @nooges

### CODE REVIEW

File: keyboards/keebio/iris/readme.md [hunk 1]
```diff
@@ -3,18 +3,36 @@ Iris
 
 A split ergo 4x6 keyboard with 3 or 4 thumb keys made and sold by Keebio. [More info at Keebio](https://keeb.io).
 
-Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)  
-Hardware Supported: Pro Micro  
-Hardware Availability: [Keebio](https://keeb.io)  
+Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)
```
Comment (fauxpark) [inline keyboards/keebio/iris/readme.md]: The "Keyboard Maintainer" and "Hardware Supported" lines need to keep the trailing double spaces, in order to render properly on GitHub.

```suggestion
Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)  
```

File: keyboards/keebio/iris/readme.md [hunk 2]
```diff
@@ -3,18 +3,36 @@ Iris
...
+Hardware Supported: Iris 3 PCB, Pro Micro, and Elite-C
```
Comment (fauxpark) [inline keyboards/keebio/iris/readme.md]:
```suggestion
Hardware Supported: Iris 3 PCB, Pro Micro, and Elite-C  
```

File: keyboards/keebio/iris/readme.md [hunk 3]
```diff
@@ -3,18 +3,36 @@ Iris
...
+
+Example of flashing this keyboard (Pro Micro):
+
     make keebio/iris/rev2:default:avrdude
 
+Example of flashing this keyboard (Elite-C):
+
+    make keebio/iris/rev2:default:dfu
+
 See [build environment setup](https://docs.qmk.fm/#/newbs_getting_started) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
```
Comment (fauxpark) [inline keyboards/keebio/iris/readme.md]: Might as well also update this line to the wording currently used in the template.

```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

**Comment by drashna** [comment]
for the flashing stuff, honestly, I'm not too keen on changing this, as I have a PR that adds the `:flash` target, which should pick the correct flasher, based on what is set as the bootloader.    Eg, it should pick `avrdude` if the bootloader is set to caterina (a stock pro micro), `dfu` if it's any type of dfu bootloader (elite C, iris rev 3, or custom flashed pro micro).

**Comment by dlrobertson** [comment]
@fauxpark thanks for the suggestions. Updated the PR.

@drashna Nice! Feel free to close the PR if that is the direction things are going in.

**Comment by drashna** [comment]
I think I'll leave it open for now.  But yeah, I think that's the direction that we should head in, for now.

**Pull request closed without merging**